src/main/kotlin/com/chattriggers/ctjs/api/world/Server.kt

```diff
@@ -53,7 +53,6 @@ object Server {
         return toMC()?.label?.let { TextComponent(it) }?.formattedText ?: ""
     }
 
-    // TODO(breaking): Return -1 if not in a world
```

Comment:
Isn't this how it's always been? It's in the ktdoc already

Reply:
It was a breaking change that wasn't documented in the migration doc, so I just added it there. On 2.2, we used to always return 5, which is what we return for singleplayer.

Reply:
I'm stupid lol I thought you added this comment, not removed it